.github/CODEOWNERS

```diff
@@ -7,23 +7,16 @@
 # be requested for review when someone opens a pull request.
 # order is alphabetical for easier maintenance.
 #
-# AnandatDell (anandrajak1)
-# Alexander Hoppe (hoppea2)
-# Alik Saring (alikdell)
 # Aaron Tye (atye)
+# Aly Nathoo (anathoodell)
 # Chiman Jain (chimanjain)
-# Don Khan (donatwork)
-# Florian Coulombel (coulof)
+# Evgeny Uglov (EvgenyUglov)
+# Harish H  (HarishH-DELL)
 # Nitesh Rewatkar (nitesh3108)
-# Sean Gallacher (gallacher)
+# Oleksandr babiychuk (alexemc)
```

Comment:
Last name should start with a capital letter.

Reply:
Fixed.